webapp/channels/src/components/pdf_preview.tsx

```diff
@@ -166,11 +168,7 @@ export default class PDFPreview extends React.PureComponent<Props, State> {
 
     getPdfDocument = async () => {
         try {
-            const PDFJS = await import('pdfjs-dist');
-            const worker = await import('pdfjs-dist/build/pdf.worker.entry.js');
-            PDFJS.GlobalWorkerOptions.workerSrc = worker;
-
-            const pdf = await PDFJS.getDocument({
+            const pdf = await pdfjsLib.getDocument({
```

Comment:
The previous version of this used a dynamic import to split the PDF library into a separate JS bundle. Can we still do that?

It might be smaller in production, but opening a PDF preview for the first time on my local web app loads about 1.4MB of JS, so we should probably keep that if we can

Reply:
@hmhealey trying the dynamic import was a PITA because after pdfjs-dist >3 they only do ESM and we would have to use import.meta.url which was causing all kind of troubles with the app and with jest, I spent enough hours trying to make it work, read tons of posts and found lot of people facing same/similar issues, the only way I made it work was this way to not have to make fancy tweaks in webpack/jest.config and babel.config, check my commits to make an idea of all the things I tried 😢

Reply:
Oh, this apparently just works. I guess we either already lazily load the `PdfPreview` component or Webpack is doing something special for the ESM file since I don't see the library loaded until I view a file attachment on the test server.

Even if nothing you tried worked, it still worked somehow 😁

Reply:
nice 🎉